reflex/utils/prerequisites.py

```diff
@@ -357,6 +357,11 @@ def update_next_config(next_config: str, config: Config) -> str:
         f'basePath: "{config.frontend_path or ""}"',
         next_config,
     )
+
+    if export:
```

Comment:
Would it be possible to read this in as a json and then do the manipulations rather than messing with the strings directly?

Reply:
sure